Feature/move config to env

KTRNJack opened the pull request.

KTRNJack committed:
feat: add .env example file

KTRNJack committed:
feat: add ignore file in git

KTRNJack committed:
feat: add ignore file in gcloud

KTRNJack committed:
feat: add function framework package

KTRNJack committed:
feat: add .nvmrc ignore

KTRNJack committed:
feat: add dotenv package

KTRNJack committed:
feat: adjust config to process env

wesley100002001 commented on service/imgur.js:
```diff
@@ -1,13 +1,20 @@
 
 exports.requestImgur = function(method, url, authKey, data, callback) {
+  let Authorization = ''
+  if ( authKey === 'auth-key') {
```
這邊用 ? 來寫吧
Authorization = authKey === 'auth-key' ?

wesley100002001 reviewed (changes requested):
改完就可以直接 merge

KTRNJack committed:
refactor: refactor codebase for consistency and clarity

- Update the Node.js runtime version to 20 for `deploy-image-dev` script in `package.json`
- Refactor `Authorization` variable assignment in `service/imgur.js` into a ternary operation.

Signed-off-by: Jack <a0972120468@gmail.com>
